### PR TITLE
Set Sri Lanka display name in country selection screen

### PR DESCRIPTION
**Pull request by msasikanth**
- Set Sri Lanka display name
- Update CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 - Migrate to Gradle Version Catalog  
 - Move logic for deciding the initial screen to the Mobius loop in `TheActivity`
 - Update OverdueListChanges feature flag to support `ContactPatientSheet` changes
+- Set Sri Lanka display name in country selection screen
```
Comment (Honey14) [inline CHANGELOG.md:13]: Shouldn't the Sri Lanka be a pointer under `Add support for Sri Lanka`?

**Review by Honey14 — CHANGES_REQUESTED** [review]
LGTM. Left one comment. PTAL

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:13]: It's not a public change yet. We haven't enabled the support for Sri Lanka right now.

Comment (Honey14) [reply CHANGELOG.md:13]: Right, and the changelog says, it's in progress. 
Wouldn't this be like :

```suggestion
- in progres Add support for Sri Lanka
   - Set Sri Lanka display name in country selection screen ```

Comment (msasikanth) [reply CHANGELOG.md:13]: Nope, it's just an internal change.

Comment (msasikanth) [reply CHANGELOG.md:13]: So basically even after we add this country support, users will not be able to see it yet. The server has to start sending us the manifest with the new country. So most of these changes will be internal.

Comment (Honey14) [reply CHANGELOG.md:13]: So Add support for Sri Lanka is also added as internal change, right? 
I know it's internal, just meant to say, this should be a pointer inside the feature. Like we started doing.

Comment (msasikanth) [reply CHANGELOG.md:13]: We are only doing that for features that require more explanation or have a vague changelog. For example things like overdue list improvements, we have added what the improvements are. In this situation we are adding support for a new country, I don't think it needs any more explanation. Once the server has finished the Sri Lanka support work, we have to coordinate the release changelog.

Comment (msasikanth) [reply CHANGELOG.md:13]: If you look at our existing changelog on master, you can see we are not adding points for every new feature. We are adding them in internal and we are only adding one-liners most of the time that are user-facing and are useful for them.

Comment (msasikanth) [reply CHANGELOG.md:13]: I have updated the changelog. I have kept the changes in internal but moved the in-progress of entry from internal to changes.

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**